src/bundle/Resources/views/content/tab/content.html.twig

```diff
@@ -1,5 +1,21 @@
+<section class="ez-fieldgroup ">
+    <h5 class="ez-fieldgroup-name" style="overflow: hidden">
+        <div class="float-left">
+        {{ 'fieldview.content'|trans({}, 'fieldview')|desc('Content') }}
```

Comment:
Why are you using `fieldview` translation domain? I can't recall we are using it anywhere in the admin UI. Location View tabs are using `locationview` domain name as far as I remember. Also you should check how we are composing translation keys in tabs.